### PR TITLE
Add openapi setup

### PR DESCRIPTION
**Pull request by b4l**

### CODE REVIEW

**Review by RemiDesgrange — APPROVED** [review]

File: scripts/setup_openapi.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+function setup_openapi() {
+  echo "Set up openapi stuff"
```
Comment (bchapuis) [inline scripts/setup_openapi.sh:4]: I would replace stuff by database ;)

File: queries/create_table_openapi.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS styles
```
Comment (bchapuis) [inline queries/create_table_openapi.sql:1]: We may add this file in the [resources](https://github.com/baremaps/baremaps/tree/main/baremaps-osm/src/main/resources) directory of baremaps. As this directory is becoming usefull in several module, we may move it elsewhere in the future. What do you think?

**Review by bchapuis — APPROVED** [review]
Looks good to me.

File: queries/create_table_openapi.sql (hunk 2 above)
Comment (b4l) [reply queries/create_table_openapi.sql:1]: The table definitions were added in https://github.com/baremaps/baremaps/pull/287/commits/9c51f77f5b6dc1ada0d6a2bb4ca27734aba370b1

File: scripts/setup_openapi.sh (hunk 1 above)
Comment (b4l) [reply scripts/setup_openapi.sh]: true, renamed it to tables

**Pull request merged**